tests/foreman/smoke/test_ui_smoke.py

```diff
@@ -314,35 +314,21 @@ def test_end_to_end(self):
                         common_locators["alert.success"]))
             # Create VM
             with VirtualMachine(distro='rhel66') as vm:
-                # Download and Install rpm
-                result = vm.run(
-                    "wget -nd -r -l1 --no-parent -A '*.noarch.rpm' "
-                    "http://{0}/pub/".format(self.server_name)
-                )
-                self.assertEqual(
-                    result.return_code, 0,
-                    "failed to fetch katello-ca rpm: {0}, return code: {1}"
-                    .format(result.stderr, result.return_code)
-                )
-                result = vm.run(
-                    'rpm -i katello-ca-consumer*.noarch.rpm'
-                )
-                self.assertEqual(
-                    result.return_code, 0,
-                    "failed to install katello-ca rpm: {0}, return code: {1}"
-                    .format(result.stderr, result.return_code)
-                )
-                # Register client with foreman server using activation-key
-                result = vm.run(
-                    'subscription-manager register --activationkey {0} '
-                    '--org {1} --force'
-                    .format(activation_key_name, org_name)
-                )
+                # Install katello cert
+                vm.install_katello_cert()
+
+                # Register client with satellite server using activation-key
+                result = vm.register_contenthost(
+                    activation_key=activation_key_name,
+                    org=org_name)
```

Comment:
Just FYI, quote from PEP8:
> The closing brace/bracket/parenthesis on multi-line constructs may either line up under the first non-whitespace character of the last line of list, [...] or it may be lined up under the first character of the line that starts the multi-line construct.

So, according to PEP8, both ways are ok. I think it's better to use the second way, as 
a) We're already following it in most cases
b) Trailing comma + closing bracket from new line will make it easier easier to add, rearrange and remove items and also may make future diffs more concise.
But that's not a blocker at all, of course.